axonn/optim.py

```diff
@@ -15,7 +15,7 @@ def __init__(
         lr=1e-3,
```

Comment:
this file should probably be called optimizer.py and we should also add an __init__.py

Reply:
Pytorch serves its optimizer as ``torch.optim.<optimizer-name>``.